<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>541</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>1</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="tab_1_plot"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>3</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>terminal</string>
     </attribute>
     <widget class="ConsoleWidget" name="terminal_widget" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>521</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>output</string>
     </attribute>
     <widget class="QTextEdit" name="output">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>481</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>461</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Data </string>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>copy data file(s) locally</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>431</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Current directory</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>461</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>461</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Model settings</string>
    </property>
    <widget class="QCheckBox" name="obs_after_HARPS_fibers_check">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>obs after HARPS fibers</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="GP_check">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>GP</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="MA_check">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MA</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="hyperpriors_check">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>hyperpriors</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="trend_check">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>trend</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="known_object_check">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>known_object</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>171</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCheckBox" name="fix_Np_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>fixed Np</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_max_Np">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>59</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>max Np</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="max_Np">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_priors">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>461</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Priors</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_slope_prior">
        <item>
         <property name="text">
          <string>Uniform</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gaussian</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_slope_prior">
        <property name="text">
         <string>trend slope</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_slope_prior_arg1"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_Cprior">
        <item>
         <property name="text">
          <string>Uniform</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gaussian</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_Cprior_arg1"/>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="makeDefault_slope_prior">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;change to default prior&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEdit_slope_prior_arg2"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_Cprior_arg2"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_Cprior">
        <property name="text">
         <string>systemic velocity</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="makeDefault_Cprior">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;change to default prior&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_Jprior_2">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>jitter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_Cprior_2">
        <item>
         <property name="text">
          <string>Uniform</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gaussian</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_Jprior_arg1"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_Jprior_arg2"/>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="makeDefault_Jprior">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;change to default prior&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>620</y>
      <width>561</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Sampler options</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_threads">
        <property name="text">
         <string>threads</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="threads">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>make plots</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>all</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>540</y>
      <width>171</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="runButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
     <property name="icon">
      <iconset theme="start">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="icon">
      <iconset theme="start">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>740</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save model</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionLoad_file"/>
    <addaction name="actionLoad_multiple_files"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuData"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_file">
   <property name="text">
    <string>Load file</string>
   </property>
  </action>
  <action name="actionLoad_multiple_files">
   <property name="text">
    <string>Load multiple files</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConsoleWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.console</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>makePlot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>677</x>
     <y>620</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>772</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>700</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>makeDefault_Cprior</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setDefaultPrior()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>692</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>makeDefault_slope_prior</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setDefaultPrior()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>674</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trend_check</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>toggleTrend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>makeDefault_Jprior</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setDefaultPrior()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>701</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>883</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
  <slot>makePlot1()</slot>
  <slot>saveModel()</slot>
  <slot>setDefaultPrior()</slot>
  <slot>toggleTrend()</slot>
  <slot>run()</slot>
 </slots>
</ui>
